### ISSUE TITLE
Finding concordances for items in inventory display

### ISSUE THREAD

**Issue by juliacmcf** [diamm/diamm#261]
Would it be possible to have a 'click here to see concordances for this work' on the tab that opens when you click on a piece in an inventory? I wonder if people realise they can get that information? It may be unnecessary of course, but might be useful, particulalry if it only appears if there ARE concordances. Ignore this if you think it's a stupid idea!
![screen shot 2017-07-20 at 10 19 12](https://user-images.githubusercontent.com/10816169/28410229-e9dfddbe-6d34-11e7-8935-1e3305cb8d5d.png)

**Comment by ahankinson** [comment]
No, I don't think it's a stupid idea. Clicking on the title "Entre je suis..." will bring you to the composition page, which shows you the concordances, but this isn't super obvious.

**Comment by juliacmcf** [comment]
Looking at this again, I've been wondering what would be useful in the 'opened' view for a composition. At the moment I think we have bibliography and comments, though comments has some bugs. If I were to choose the most important things, they would be concordances and comments (since comments should say if the work is just a fragment). But then we have no way to show bibliography for an item unless it shows under the composition (via clicking on the title). I'm assuming incidentally that bibliography for compositions is one of the things that would appear when you click on the title -- I haven't found a composition to test where I know there should be a bibliog.

So it would save clicking and going through unnecessary windows if we could have the concordances showing in the 'open' view within the inventory.

Another more labour-intensive option would be to have links on the right for Comments/Concordances/Bibliography which generate a popup on mouseover (or click), so you don't have to leave the master inventory to see them.

**Comment by ahankinson** [comment]
I think I'm going to close this.I had a go at adding the concordances to the inventory but there were two problems:

1) It made the queries much more complex 
2) It caused the inventory display to have way too much information

For now I think people will have to satisfy themselves with seeing the concordance information on the composition screen.
